cli/cobra.go

```diff
@@ -205,6 +208,27 @@ func isExperimental(cmd *cobra.Command) bool {
 	return experimental
 }
 
+func additionalHelp(cmd *cobra.Command) string {
+	if additionalHelp, ok := cmd.Annotations["additionalHelp"]; ok {
+		return additionalHelp
+	}
+	return ""
+}
+
+func hasAdditionalHelp(cmd *cobra.Command) bool {
+	return additionalHelp(cmd) != ""
+}
+
+var cyan = color("\033[1;36m%s\033[0m")
```

Comment:
@nebuk89 can you confirm what is the right color? I think the spec said "light cyan" which might be "bright cyan" [here](https://en.wikipedia.org/wiki/ANSI_escape_code#Colors), i.e.: 96

Reply:
Ah yes I meant bright sorry!

Reply:
Updated

Reply:
updated